databricks/sdk/mixins/compute.py

```diff
@@ -231,6 +232,11 @@ def ensure_cluster_is_running(self, cluster_id: str) -> None:
                     return
                 elif info.state in (state.ERROR, state.UNKNOWN):
                     raise RuntimeError(f'Cluster {info.cluster_name} is {info.state}: {info.state_message}')
+            except DatabricksError as e:
+                if e.error_code == 'INVALID_STATE':
+                    _LOG.debug(f'Cluster was started by other process: {e} Retrying.')
+                    continue
```

Comment:
It might make sense to retry only once/limited number of times. Because who knows what are the other errors can go from cluster-manager with this error code. And if there is something else, it will be an infinite loop

Reply:
we retry this loop with a timeout, look at the beginning of the method